### PR TITLE
Dev pydantic rdflib

### PR DESCRIPTION
**Pull request by jb2197**
This PR provides an implementation of Object Graph Mapper in Python by connecting `pydantic` and `rdflib`.

### CODE REVIEW

File: JPS_BASE_LIB/python_wrapper/README.md
Comment (sm453) [inline JPS_BASE_LIB/python_wrapper/README.md:1]: It's a bit hard to see in this 'diff forest', but it appears as if you largely removed all previous documentation of the former `py4jps`? Surely this hasn't all become obsolete?

**Review by sm453 — CHANGES_REQUESTED** [review]

File: JPS_BASE_LIB/python_wrapper/tests/conftest.py [hunk 1]
```diff
@@ -20,17 +23,33 @@ def initialise_triple_store():
...
 
     with blazegraph as container:
-        # Wait some arbitrary time until container is reachable
-        time.sleep(3)
-
-        # Retrieve SPARQL endpoint
-        endpoint = get_endpoint(container)
+        # It ensures that the requested Blazegraph Docker service is ready to accept SPARQL query/update
+        service_available = False
+        while not service_available:
```
[the remark this answers is not in the document]
Comment (jb2197) [reply JPS_BASE_LIB/python_wrapper/tests/conftest.py]: Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1133/commits/97b02d29c38a4bf6f1663f0b8943e387850d0162.

File: JPS_BASE_LIB/python_wrapper/README.md
Comment (jb2197) [reply JPS_BASE_LIB/python_wrapper/README.md:1]: The previous documentations are now moved to relevant files in `python_wrapper/docs/examples` that will be accessed on the `twa` docs webpage, the idea was to keep the readme file minimal.

File: JPS_BASE_LIB/python_wrapper/twa/__init__.py [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+    TransitiveProperty,
+    as_range,
+)
+from twa.kg_operations import (
+    PySparqlClient,
+)
+from twa.data_model.iris import TWA_BASE_URL
 
-__version__ = "0.0.1"
+__version__ = "0.0.2a0"
```
[the remark this answers is not in the document]
Comment (jb2197) [reply JPS_BASE_LIB/python_wrapper/twa/__init__.py]: Addressed in https://github.com/cambridge-cares/TheWorldAvatar/pull/1133/commits/7e1ad7cebc8b2475a6b437bda5ebf92e9c0fbb2c. Also made a release of [`twa==0.0.2`](https://pypi.org/project/twa/0.0.2/).

**Review by sm453 — APPROVED** [review]
Design discussed extensively during the whole project. Any further developments to be undertaken on separate branches.

**Pull request merged**